app/routes.py

```diff
@@ -21,11 +21,16 @@
 from bs4 import BeautifulSoup as bsoup
 from flask import jsonify, make_response, request, redirect, render_template, \
     send_file, session, url_for
-from requests import exceptions
+from requests import exceptions, get
 
 # Load DDG bang json files only on init
 bang_json = json.load(open(app.config['BANG_FILE']))
 
+# Get the newest version of WHOOGLE
+link_releases = "https://github.com/benbusby/whoogle-search/releases"
+update = bsoup(get(link_releases).text, "html.parser")
```

Comment:
Small change here -- I try to uniformly use single quote strings throughout the repo

```suggestion
update = bsoup(get(link_releases).text, 'html.parser')
```

This will also have to be updated to reflect the change noted above as well